napari/utils/misc.py

```diff
@@ -441,3 +449,55 @@ def dir_hash(path: Union[str, Path], include_paths=True, ignore_hidden=True):
                 _hash.update(''.join(fparts).encode())
 
     return _hash.hexdigest()
+
+
+def _generate_cls_stubs(cls, output_file, imports: Sequence[str] = ()) -> str:
+    """Generate pyi-format type stubs for a class."""
+    import textwrap
+
+    from ..utils.misc import get_subclass_methods
+
+    bases = ", ".join(f'{b.__module__}.{b.__name__}' for b in cls.__bases__)
+
+    pyi = '# THIS FILE IS AUTOGENERATED.  DO NOT EDIT\n'
```

Comment:
could you also add a comment saying that the code to generate it can be found here?